indolaw-parser/parser_main.py

```diff
@@ -1842,11 +1842,17 @@ def crash(law: List[str], i: int, error_message: str) -> None:
     parse_undang_undang(ROOT, law)
 
     assert ROOT is not None
+
+    metadata = extract_metadata_from_tree(ROOT)
+
+    ketentuan_umum_list = sorted(metadata['ketentuan_umum'].keys(), key=lambda x: len(x), reverse=True)
+    content = convert_tree_to_json(ROOT, ketentuan_umum_list)
+
     with open(filename + '.json', 'w') as outfile:
         json.dump(
             {
-                'metadata': extract_metadata_from_tree(ROOT),
```

Comment:
Btw @JohnAmadeo, any reason why the order of the json (content / metadata) is reversed?

Reply:
So that if I want to see metadata in the JSON I don't have to scroll all the way down to the bottom of the file hahahaha